app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -13,10 +13,7 @@ import android.view.Menu
 import android.view.MenuItem
 import android.view.View
 import android.view.inputmethod.EditorInfo
-import android.widget.Button
-import android.widget.ImageButton
-import android.widget.RadioButton
-import android.widget.Toast
+import android.widget.*
```

Comment:
Prefer single class imports over * imports.

You can configure this in Android Studio under File > Settings > Editor > Code Style > Kotlin > Imports.